test/e2e/dist-mnist/tj_job_mnist.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: "kubeflow.org/v1alpha2"
+kind: "TFJob"
+metadata:
+  name: "dist-mnist-for-e2e-test"
+spec:
+  tfReplicaSpecs:
+    PS:
+      replicas: 2
+      restartPolicy: Never
+      template:
+        spec:
+          containers:
+            - name: dist-mnist-ps
+              image: kubeflow/tf-dist-mnist-test:1.0
```

Comment:
Can we put this image in google registry?

Reply:
I think so, while it is a separate issue.

Reply:
Do you mean doing this in another PR?

Reply:
See #590 I think we need help from @jlewi 

I do not have the permission to push the image to google or dockerhub registry.